src/scene/minigame_scenes/mgscene_penguin.cpp

```diff
@@ -17,19 +18,24 @@ void MGScenePenguin::s_setup() {
     // Set up penguins. Initially disabled.
     for (int i = 0; i < MAX_PENGUINS; i++) {
         Penguin* to_add = new Penguin();
+
+        // Set spawn position and rotation
+        to_add->set_position(glm::vec3(10, 1, rand() % 10 - 5));
```

Comment:
Probably a bit cleaner to just add these as constructor parameters, but meh whatever